frontend/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+import ProfileModal from "./components/ProfileModal";
+import LeftSection from "./components/LeftSection";
+import RightSection from "./components/RightSection";
+
+interface HeaderProps {
+  lecture?: boolean;
+  login?: boolean;
+  main?: boolean;
+}
+
+const Header: React.FC<HeaderProps> = ({ lecture, login, main }) => {
```

Comment:
React.FC는 사용하지 않는 추세라는 글을 오늘 읽은 적이 있어요. 아래 처럼 수정하는 편이 코드 길이도 줄어들고 가독성도 좋은 것 같습니다!
```suggestion
const Header = ({ lecture, login, main }:HeaderProps) => {
```

Reply:
헉! 이 부분은 몰랐네요! 바로 반영해보겠습니다.